x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/execute.ts

```diff
@@ -493,6 +500,51 @@ export default function ({ getService }: FtrProviderContext) {
               throw new Error(`Scenario untested: ${JSON.stringify(scenario)}`);
           }
         });
+
+        it('should authorize system actions correctly', async () => {
+          const connectorId = 'system-connector-test.system-action-kibana-privileges';
+          const name = 'System action: test.system-action-kibana-privileges';
+
+          const response = await supertestWithoutAuth
+            .post(`${getUrlPrefix(space.id)}/api/actions/connector/${connectorId}/_execute`)
+            .auth(user.username, user.password)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              params: {},
+            });
+
+          switch (scenario.id) {
+            case 'no_kibana_privileges at space1':
+            case 'space_1_all_alerts_none_actions at space1':
+            case 'space_1_all at space2':
+            case 'global_read at space1':
+            case 'space_1_all at space1':
+            case 'space_1_all_with_restricted_fixture at space1':
+              expect(response.statusCode).to.eql(403);
+              expect(response.body).to.eql({
+                statusCode: 403,
+                error: 'Forbidden',
+                message: 'Unauthorized to execute actions',
+              });
+              break;
+            case 'superuser at space1':
+            case 'system_actions at space1':
```

Comment:
Only the superuser and the `cases` user have access to the system action and thus can execute the system action.  `space_1_all` et al, even though they have access to actions they do not have access to the system action so the API call should fail with `403`.